src/components/Cta/CtaLink.tsx

```diff
@@ -52,6 +52,10 @@ export const CtaLink = React.forwardRef<HTMLAnchorElement, CtaLinkProps>(
     if (isInternal) {
       myLink = cachedUrl || href;
 
+      if (!myLink.startsWith('/')) {
+        myLink = myLink === 'home' ? '/' : `/${myLink}`;
+      }
```

Comment:
This was missing which screws with scrolling to top on route change and previously these links renders as relative paths, eg, if I'm on /about and add a Storyblok link with cachedUrl to "test", it will end up linking to /about/test, which isn't what we want. We want it to link to "/test".